index.html

```diff
@@ -61,9 +62,204 @@ <h3>LET'S CONNECT</h3>
+
+                        </div>
+                        <div class="tonic">
+                            <h2>Tonic</h2>
+                        </div>
+                        <div class="frame">
+                            <img src="images.jpeng/Frame2.png" alt="">
+
+                        </div>
+                        <div class="instruction">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                        </div>
+                        <div class="categorie-work">
+                            <ul>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Taghtml.png" alt=""></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Tagcss.png" alt=""></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Tagjavascript.png" alt=""></a>
+                                </li>
+                            </ul>
+                        </div>
+                        <button type="button" class="see-button">See Project</button>
+                    </div>
+                    <div class="work-section3">
+                        <div class="image-4">
+                            <img src="images.jpeng/SnapshootPortfolio4.png" alt="">
+
+                        </div>
+                        <div class="multi-post">
+                            <h4>Multi-Post
+                                Stories</h4>
+                        </div>
+                        <div class="frame">
+                            <img src="images.jpeng/Frame2.png" alt="">
+
+                        </div>
+                        <div class="instruction">
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                        </div>
+                        <div class="categorie-work">
+                            <ul>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Taghtml.png" alt=""></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Tagcss.png" alt=""></a>
+                                </li>
+                                <li>
+                                    <a href="#"><img src="images.jpeng/Tagjavascript.png" alt=""></a>
+                                </li>
+                            </ul>
+                        </div>
+                        <button type="button" class="see-button">See Project</button>
+                    </div>
+                </section>
+                    <!-- about -->
+                    <div class="about-wrapper">
```

Comment:
- [ ] As the previous code reviewer suggested. I kindly suggest that you use a `section` tag here instead of a `div` tag

### EXAMPLE
:x: Wrong
`<div class="about-wrapper">
`

:heavy_check_mark: Right
`<section class="about-wrapper">
`